src/components/pages/Navbar/Navbar.js

```diff
@@ -67,6 +84,30 @@ const Navbar = ({ isAuthenticated, userProfile, getProfile }) => {
     // push(menu.key);
   };
 
+  const handleToggleChange = checked => {
+    if (!checked) {
+      axiosWithAuth()
+        .post(`/profile/availability/${profile_id}`, {
+          accepting_new_mentees: false,
+        })
+        .then(res => {
+          setToggleStatus(false);
+        })
+        .catch(err => console.log(err));
```

Comment:
same comment about error handling from above...
Love this function, though! super straightforward.